src/css/custom.css

```diff
@@ -0,0 +1,45 @@
+/**
+ * Any CSS included here will be global. The classic template
+ * bundles Infima by default. Infima is a CSS framework designed to
+ * work well for content-centric websites.
+ */
```

Comment:
Personally I would prefer to use `tailwind` over this if we have to redo styles either way, as it supports JIT compiling and it's one of the most popular and well maintained ones (and I'm using it on all my latest projects so I could make changes much faster in this repo then too)